pkg/chart/chart_test.go

```diff
@@ -216,19 +216,47 @@ func TestParseDetailsForHTTPClient(t *testing.T) {
 	)
 
 	testCases := []struct {
-		name             string
-		details          *Details
-		appRepoSpec      appRepov1.AppRepositorySpec
-		errorExpected    bool
-		numCertsExpected int
+		name              string
+		details           *Details
+		appRepoSpec       appRepov1.AppRepositorySpec
+		errorExpected     bool
+		numCertsExpected  int
+		cluster           string
+		kubeappsNamespace string
+		kubeappsCluster   string
 	}{
+		{
+			name: "default cert pool without auth (as svc)",
```

Comment:
I think our heads are in this now, so we understand, but in 3 months (or to someone else) more context will be useful?
```suggestion
			name: "default cert pool without auth (as svc because cluster and kubeappsCluster are empty)",
```